README.md

```diff
@@ -107,6 +107,22 @@ If not one of the well-described tests, the Fake Provider exhibits the following
 - If the CPR number ends in `000`, `001` or `002` it returns NotFound
 - The Fake provider returns random data, using the CPR number as the seed. This ensures recurring calls return the same response
 
+## CPR Subscription
+
+Whenever there is a change in information about a Citizen the CPR provider will raise events.
```

Comment:
We should probably clarify here:

The events raised will include citizens you have no records for. You can either:
a. Call `GetAllCprEvents` and determine the relevant events yourself
b. Subscribe for events about specific citizens then call `GetSubscribedCprEvents` to return just those you are interested in

Reply:
changed as asuggested.